gin.go

```diff
@@ -645,7 +645,7 @@ func (engine *Engine) HandleContext(c *Context) {
 
 func (engine *Engine) handleHTTPRequest(c *Context) {
 	httpMethod := c.Request.Method
-	rPath := c.Request.URL.Path
+	rPath := c.Request.URL.EscapedPath()
```

Comment:
This should not be changed right?
Now we are always routing based on the `EscapedPath` which changes the default behaviour

Reply:
> This should not be changed right? Now we are always routing based on the `EscapedPath` which changes the default behaviour

You are right.